avatars/avatars.js

```diff
@@ -1973,56 +1974,115 @@ class Avatar {
   }
 
   isAudioEnabled() {
-    return !!this.microphoneWorker;
+    return !!this.audioWorker;
   }
   setAudioEnabled(enabled) {
+    // cleanup
+    if (this.audioWorker) {
+      this.audioWorker.close();
+      this.audioWorker = null;
+    }
+
+    // setup
+    if (enabled) {
+      this.ensureAudioRecognizer();
+      this.volume = 0;
+
+      const audioContext = getAudioContext();
+      if (audioContext.state === 'suspended') {
+        (async () => {
+          await audioContext.resume();
+        })();
+      }
+      const _volume = e => {
+        if (!this.manuallySetMouth) {
```

Comment:
`manuallySetMouth` should be documented when declared. It's for chomping and attack screams.